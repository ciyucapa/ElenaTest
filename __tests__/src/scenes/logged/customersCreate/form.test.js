import React from 'react';
import {create} from 'react-test-renderer';

import Form from '../../../../../src/scenes/logged/customersCreate/form';

describe('Customer create form component', () => {
    test('render Customer create form component', () => {
        const tree = create(
            <Form
                firstName="Test"
                lastName="Lastname test"
                cellphone="+578484545"
                streetAddress="Address"
                country="test"
                isValidForm={false}
            />,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});