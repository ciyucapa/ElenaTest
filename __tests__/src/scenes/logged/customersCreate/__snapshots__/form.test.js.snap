// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer create form component render Customer create form component 1`] = `
<div
  className="sc-gKseQn fqbNkd"
>
  <input
    onChange={[Function]}
    placeholder="Nombres"
    type="text"
    value="Test"
  />
  <input
    onChange={[Function]}
    placeholder="Apellidos"
    type="text"
    value="Lastname test"
  />
  <input
    onChange={[Function]}
    placeholder="Telefono"
    type="text"
    value="+578484545"
  />
  <select
    onChange={[Function]}
    value=""
  />
  <input
    onChange={[Function]}
    placeholder="DirecciÃ³n"
    type="text"
    value="Address"
  />
  <input
    onChange={[Function]}
    placeholder="Pais"
    type="text"
    value="test"
  />
  <button
    className="sc-iBPTik xaJDg"
    disabled={true}
    onClick={[Function]}
  >
    Enviar
  </button>
</div>
`;
